pom.xml

```diff
@@ -3,11 +3,14 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>net.i2p.crypto</groupId>
   <artifactId>eddsa</artifactId>
-  <version>0.1.0</version>
+  <version>0.1.1</version>
```

Comment:
I have some other changes I'll be bundling in, so please revert the version for this PR.